ex15/index.js

```diff
@@ -0,0 +1,45 @@
+const prompt = require("prompt-sync")({ sigint: true});
+
+let decimalNumbers = [];
+let weightOfNumbers = [];
+let input;
+let inputWeight;
+
+while( input !== 0 && inputWeight !== 0) {
+    console.log("Insert an decimal number (use 0 to stop de loop):");
+    input = parseFloat(prompt(">"));
+
+    if (input === 0) {
+        break;
+    }
+
+    console.log("Insert the weight of this number (use 0 to stop de loop):");
+    inputWeight = Number(prompt(">"));
+
+    if (inputWeight === 0) {
+        break;
+    } 
+
+    decimalNumbers.push(input);
+    weightOfNumbers.push(inputWeight); 
+} 
+
+function weightedArithmeticMean () {
+
+    let sumOfTheTop = 0
+    let sumOfTheBottom = 0;
+    
+
+    for(let i = 0; i < decimalNumbers.length; i++) {
+        const topMultiply = decimalNumbers[i] * weightOfNumbers[i];
+        sumOfTheTop = sumOfTheTop + topMultiply;
+        sumOfTheBottom = sumOfTheBottom + weightOfNumbers[i]; 
+
+    }
+    
+    let finalResult = sumOfTheTop / sumOfTheBottom
+    return finalResult;
```

Comment:
this could be a const (since it's not re-assigned):
```js
    const finalResult = sumOfTheTop / sumOfTheBottom
    return finalResult;
```

Or simply:
```js
    return sumOfTheTop / sumOfTheBottom;
```